cmd/dep/ensure.go

```diff
@@ -692,7 +685,8 @@ func (cmd *ensureCommand) runAdd(ctx *dep.Ctx, args []string, p *dep.Project, sm
 		}
 	}
 
-	return errors.Wrapf(f.Close(), "closing %s", dep.ManifestName)
+	//return errors.Wrapf(f.Close(), "closing %s", dep.ManifestName)
```

Comment:
no need to change this line - passing a nil error to `errors.Wrapf()` will end up passing back a nil.